src/style.css

```diff
@@ -0,0 +1,90 @@
+/* stylelint-disable color-hex-length */
+/* stylelint-disable color-hex-case */
+@font-face {
+  font-family: Avenir;
+  font-weight: 400;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-family: Avenir, Helvetica, sans-serif;
+  font-weight: 300;
+}
+
+.bigOne {
+  display: block;
+  margin-top: 20px;
+}
+
+.smallOne {
+  display: block;
+  margin-top: 20px;
+}
+
+.bigBar {
+  padding-left: 62px;
+  font-family: Avenir, Helvetica, sans-serif;
+  font-weight: 900;
+  font-size: 16px;
+  background-image: url(images/Search.svg);
+  height: 70px;
+  width: 100%;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  background-repeat: no-repeat;
+  background-position: 26px;
+  background-size: 19px;
+  outline: none;
+}
+
+.bigBar:hover {
+  box-shadow: 0 3px 8px 0 #3d4e6133;
+}
+
+.bigBar::placeholder {
+  font-weight: 400;
+}
+
+.bigBar:focus {
+  background: url(images/Search.svg) no-repeat, linear-gradient(180deg, #ffffff 0%, #f6f6f7 100%);
+  background-position: 26px;
+  background-size: 19px;
+}
+
+.smallBar {
+  padding-left: 33px;
+  font-family: Avenir, Helvetica, sans-serif;
+  font-weight: 900;
+  font-size: 14px;
+  background-image: url(images/Search.svg);
+  height: 42px;
+  width: 100%;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
```

Comment:
You have a lot of repeated rules here (same as in `bigBar`)
Use `,` grouping selector